lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -575,6 +575,12 @@ sub add_api_user {
         if ( $allow ) {
             $result = 1 if ( $self->{db}->add_api_user( $params->{username}, $params->{api_key} ) eq '1' );
         }
+        else {
+            die Zonemaster::Backend::Error::PermissionDenied->new(
+                message => 'Unauthorized to call this method from a remote IP',
```

Comment:
I think that "forbidden" or "not permitted" is better than "unauthorized" in this context. There is no authorization mechanism for this. Suggestions:
```
'Call to this method from a remote IP is not permitted'
```

Reply:
Changed to:
```
Call to "add_api_user" method not permitted from a remote IP
```